### ISSUE TITLE
Custom embed from .sb3 file url

### ISSUE THREAD

**Issue by ghost** [LLK/scratch-gui#6103]
I'm trying to create my own embed scratch to allow user to play a file only without the code blocks.

I already custom the gui to let my user save and open files in it to modify the code. but i now want to create something to make the user play the games created full screen like 'https://scratch.mit.edu/projects/NUM_PROJ/fullscreen/' or 'https://scratch.mit.edu/projects/NUM_PROJ/embed/' from a url.

I'm not sure if i need to tweak scratch-vm or only scratch-render, or if i'll need the full scratch-gui to load a file.

if you guys know and can clarify me what i should use to do that it would be great.

basically i want to do a custom embed project...

**Comment by apple502j** [comment]
You need full scratch-gui to render it perfectly, but scratch-vm is enough for some cases (when you don't render and just want to run as background)
For loading from URL maybe check https://github.com/LLK/scratch-gui/blob/develop/src/lib/hash-parser-hoc.jsx
